src/components/CandidateAvatar/CandidateAvatar.tsx

```diff
@@ -45,7 +45,7 @@ export const CandidateAvatar: React.FC<Props> = ({
     return (
         <div
             {...rest}
-            {...block()}
+            {...block({ ...rest })}
```

Comment:
there are usage of `elem` here where you removed the props (looking at the previous PR) but you haven't added it back